Client/Frontend/Home/RemoteTabsPanel.swift

```diff
@@ -518,3 +552,72 @@ class RemoteTabsNotLoggedInCell: UITableViewCell {
         super.updateConstraints()
     }
 }
+
+private class HistoryTableViewHeader : UIView {
```

Comment:
Nit: If it's not a `UITableViewFooterHeaderView` subclass, drop the `TableViewHeader` and use just `View`